server.R

```diff
@@ -142,14 +142,22 @@ server <- function(input, output, session) {
   selection <- reactive({
     stat_hq %>%
       filter(Region == input$region &
-        Sector == input$sector)
+        Sector == input$sector) %>%
+      mutate(Level = case_when(
+        Level_order == "Below level 2" ~ "Below level 2 (GCSE grades 1-3)",
+        Level_order == "Level 2" ~ "Level 2 (GCSE grades 4-9)",
+        Level_order == "Level 3" ~ "Level 3 (GCE A level)",
+        Level_order == "Level 4/5" ~ "Level 4/5 (HNC or HND)",
+        Level_order == "Level 6" ~ "Level 6 (Degree)",
+        TRUE ~ "Level 7 (Master's degree)"
```

Comment:
Can you call this "Level 7+ (Master's degree) - we grouped up L7 and L8 as there are a small number of PHDs in the data

Reply:
ok